pkg/packageinstall/packageinstall.go

```diff
@@ -241,17 +241,28 @@ func (pi *PackageInstallCR) update(updateFunc func(*pkgingv1alpha1.PackageInstal
 	pi.log.Info("Updating installed package")
 
 	modelForUpdate := pi.model.DeepCopy()
-	updateFunc(modelForUpdate)
 
-	var err error
+	var lastErr error
+	for i := 0; i < 5; i++ {
```

Comment:
Any reason for 5 retries?

Reply:
No, it's just the number we went with last time we ran in to this